modules/ROOT/pages/users.adoc

```diff
@@ -86,6 +86,14 @@ include::partial$include-nav-users.adoc[]
 
 The user now has the permissions you selected in the business groups and environments you specified.
 
+== What happens after a user is deleted
+
+After a user is deleted from the top level Organization, the user is "soft-deleted" and deasocited with the Organization. The reason why it's not "hard-deleted" is because the user identity is referenced in the Audit Log. 
```

Comment:
```suggestion
When you delete a user from your root organization, the user is _soft deleted_, wherein a user can no longer access the root organization or business groups and becomes an orphaned Anypoint Platform account unless associated with another organization. Anypoint Platform maintains the user's former presence in the organization for the purposes of audit logs. 
```

Reply:
suggest "dangling" instead of "orphaned" (more sensitive)

Reply:
I prefer to change "user's former presence in the organization" -> user's former presence. Removing "the organization" because the user is disassociated from the org.

Reply:
Changed just a bit more @ductran-ms